docs/installation.md

```diff
@@ -0,0 +1,61 @@
+## Installation
+
+Following ways are supported to run Kraken:
+
+- Standalone python program through Git
+- Containerized version using either Podman or Docker as the runtime
+- Kubernetes or OpenShift deployment
+
+**NOTE**: It is recommended to run Kraken external to the cluster ( Standalone or Containerized ) hitting the Kubernetes/OpenShift API as running it internal to the cluster might be disruptive to itself and also might not report back the results if the chaos leads to cluster's API server instability.
+
+### Git
+
+#### Clone the repository
+```
+$ git clone https://github.com/openshift-scale/kraken.git
+```
+
```

Comment:
Kind of a given here but I have read a lot of documentation that even says `cd kraken` thoughts on adding this here?

Reply:
Done.